src/sys/stat.rs

```diff
@@ -122,3 +122,48 @@ pub fn fstatat<P: ?Sized + NixPath>(dirfd: RawFd, pathname: &P, f: AtFlags) -> R
     Ok(dst)
 }
 
+/// Change the file permission bits of the file named by the pathname.
+///
+/// # References
+///
+/// [chmod(2)](http://pubs.opengroup.org/onlinepubs/9699919799/functions/chmod.html).
+pub fn chmod<P: ?Sized + NixPath>(path: &P, mode: Mode) -> Result<()> {
+    let res =
+        path.with_nix_path(|cstr| unsafe { libc::chmod(cstr.as_ptr(), mode.bits() as mode_t) })?;
+
+    Errno::result(res).map(|_| ())
+}
+
+/// Change the file permission bits of the file specified by the file descriptor.
```

Comment:
"Change the file permission bits of the file specified by the file descriptor." -> "Change the file permission bits of the file specified by a file descriptor"